packages/components/input/__tests__/input.test.tsx

```diff
@@ -37,6 +37,14 @@ describe("Input", () => {
     expect(container.querySelector("input")).toHaveAttribute("disabled");
   });
 
+  it("should set tabIndex to -1 when isDisabled is true", () => {
```

Comment:
The test should verify that clicking the button does not clear the input.

Reply:
![image](https://github.com/user-attachments/assets/4bf8f900-7d59-460b-bd84-1eb7f4ddffde)

The clear button is already non-functional when isDisabled is true.

Reply:
That’s true, but if you’re adding tests, I would appreciate it if you could verify that the button is disabled. 
I think specifying the `tabindex` will be resolved by marking it up as a `button`.